### PR TITLE
Release B3

### PR DESCRIPTION
**Pull request by sraturi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@opensea/seaport-js": "^4.0.0",
-    "ethers": "^6.9.0"
+    "ethers": "^6.9.0",
+    "opensea-js": "^7.1.13"
```
Comment (jalagar) [inline package.json:43]: why is this added?

**Review by jalagar — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sraturi) [reply package.json:43]: based on https://www.notion.so/opensea/EPD-Supporting-a-new-chain-7b4f81c6dd26413eb784db78f181afd0?pvs=4#7a76b961bef74121bf6bfe2b9c086f76

**Review by jtaaa — CHANGES_REQUESTED** [review]
Should not add opensea-js to deps

File: package.json (hunk 1 above)
Comment (jtaaa) [reply package.json:43]: need to bump the version above. add to deps here

**Review by jtaaa — APPROVED** [review]

**Comment by its2badwespend** [comment]
> 
> 
> ## Motivation
> 
> 
> 
> ## Solution
> 
> 
>
